data/gmh/Brackert_1963_Nibelungenlied/metadata.txt

```diff
@@ -0,0 +1,34 @@
+publicationType : "book"
+publicationTitle : "Beiträge zur Handschriftenkritik des Nibelungenliedes"
+publicationDate : "1963"
+publicationPlace : "Berlin"
+publicationSeries : " Quellen und Forschungen zur Sprach- und Kulturgeschichte der germanischen Völker. N.F."
+publicationNum: "11"
+publicationStemmaNum: ""
+publicationAuthors : "Brackert, Helmut"
+publicationPage : "174"
+publicationLink : "https://www.degruyter.com/view/title/6743"
+workTitle : "Nibelungenlied"
+workViaf : "181844575"
+workOrigDate : "1200-1205"
+workOrigPlace : "Passau(?)"
+workAuthor : "Anonymous"
+workAuthorViaf : ""
+workGenre : "Heldendichtung"
+workLangCode : "gmh"
+stemmaType : "reconstructed"
+contam : "no"
```

Comment:
I believe there is, though (duplicated mss as you said).

Reply:
Right